src/mysky/encrypted_files.test.ts

```diff
@@ -48,46 +48,75 @@ describe("deriveEncryptedFileKeyEntropy", () => {
       94, 186, 244, 48, 171, 115, 171,
     ];
 
-    const result = deriveEncryptedFileKeyEntropy(pathSeed);
+    const result = deriveEncryptedPathKeyEntropy(pathSeed);
 
     expect(result).toEqualUint8Array(new Uint8Array(expectedEntropy));
   });
 });
 
-describe("deriveEncryptedFileSeed", () => {
-  it("Should derive the correct encrypted file seed", () => {
-    // Hard-code expected value to catch breaking changes.
-    const pathSeed = "a".repeat(64);
-    const subPath = "path/to/file.json";
+describe("deriveEncryptedPathSeed", () => {
+  // Hard-code expected value to catch breaking changes.
+  const pathSeed = "a".repeat(128);
+  const subPath = "path/to/file.json";
+  const expectedDirectorySeed =
+    "ae6e5be22469f4c6d89d8490906207d70753a06072db3af44c66e35b37c968a2c5457417f1cf099e3630e8255a58208d6a4b32a174c56c74877fef6412d1a02d";
+  const expectedFileSeed = "40fb1701ab23fb020358baa5d5a2cba97d2de64c1e1418382ee6623fd9c17995";
 
+  it("should derive the correct encrypted file seed for a file", () => {
     // Derive seed for a file.
-    const fileSeed = deriveEncryptedFileSeed(pathSeed, subPath, false);
+    const fileSeed = deriveEncryptedPathSeed(pathSeed, subPath, false);
 
-    expect(fileSeed).toEqual("ace80613629a4049386b3007c17aa9aa2a7f86a7649326c03d56eb40df23593b");
+    expect(fileSeed).toEqual(expectedFileSeed);
+  });
 
+  it("should derive the correct encrypted file seed for a directory", () => {
     // Derive seed for a directory.
-    const directorySeed = deriveEncryptedFileSeed(pathSeed, subPath, true);
+    const directorySeed = deriveEncryptedPathSeed(pathSeed, subPath, true);
+
+    expect(directorySeed).toEqual(expectedDirectorySeed);
+  });
 
-    expect(directorySeed).toEqual("fa91607af922c9e57d794b7980e550fb15db99e62960fb0908b0f5af10afaf16");
+  it("should result in the same path seed when deriving path for directory first", () => {
+    // Derive seed for directory first.
+    const directorySeed = deriveEncryptedPathSeed(pathSeed, "path/to", true);
 
-    expect(fileSeed).not.toEqual(directorySeed);
+    // Derive seed for file.
+    const fileSeed = deriveEncryptedPathSeed(directorySeed, "file.json", false);
+
+    expect(fileSeed).toEqual(expectedFileSeed);
```

Comment:
Is the regression test? I think it makes sense to copy-paste the PR description here somewhere, perhaps a bit more concise, and very clearly label a test as the regression test. It's always could to have a small docstring above a unit test that very clearly labels it a regression test and explain what the regression was and how it was handled.